drafts/log4j-rhocp.adoc

```diff
@@ -226,14 +227,8 @@ The following log is an example output:
 {"type":"liberty_message","host":"192.168.0.104","ibm_userDir":"\/Users\/yushan.lin@ibm.com\/Documents\/archived-guide-log4j\/finish\/target\/liberty\/wlp\/usr\/","ibm_serverName":"log4j.sampleServer","message":"hello liberty servlet warning message!","ibm_threadId":"00000035","ibm_datetime":"2020-02-13T11:27:07.792-0500","module":"application.servlet.LibertyServlet","loglevel":"WARNING","ibm_methodName":"doGet","ibm_className":"application.servlet.LibertyServlet","ibm_sequence":"1581611227792_0000000000017","ext_thread":"Default Executor-thread-8"}
 ```
 
-As you've seen in this post, there are different ways to send your Log4j 2 and other non-JUL logs to the RHOCP EFK stack. You can use a sidecar container to stream logs directly to stdout, or you can run the sidecar container as your logging agent. You can also implement the Log4j 2 to SLF4J Adapter to merge Log4j 2 logs with your Liberty logs and output them in JSON format.
-
-**Related links**
-
-* link:https://openliberty.io/docs/ref/general/#logging.html[Open Liberty logging documentation]
-
-* link:https://developer.ibm.com/videos/use-json-logging-in-open-liberty[Use JSON logging in Open Liberty]
+If you want to learn more about JSON logging with Open Liberty, check out this link:https://developer.ibm.com/videos/use-json-logging-in-open-liberty[short YouTube video] or this blog post about link:https://openliberty.io/blog/2019/12/03/custom-fields-json-logs.html[Adding custom fields to JSON logs in Open Liberty].
 
-* link:https://openliberty.io/blog/2019/12/03/custom-fields-json-logs.html[Adding custom fields to JSON logs in Open Liberty]
+== Summary
 
-* link:https://community.ibm.com/community/user/imwuc/viewdocument/on-demand-using-liberty-with-elast[Using Liberty with Elastic Stack (aka ELK)]
+As you've seen in this post, there are different ways to forward your Log4j 2 and other non-JUL logs to RHOCP's EFK stack. You can use a sidecar container to stream logs directly to stdout, or you can run the sidecar container as your logging agent. You can also implement the Log4j 2 to SLF4J Adapter to merge Log4j 2 logs with your Liberty logs and output them in JSON format. For more information about logging in Open Liberty, see the link:https://openliberty.io/docs/ref/general/#logging.html[Open Liberty logging documentation]. If you want to try out another step-by-step tutorial on logging in EFK, check out this link:https://kabanero.io/guides/app-logging-ocp-4-2/[Kabanero guide on Application Logging on RHOCP]. Or, if you want to learn about Open Liberty and the ELK (Elasticsearch, Logstash, Kibana) stack, check out link:https://community.ibm.com/community/user/imwuc/viewdocument/on-demand-using-liberty-with-elast[this video on Using Liberty with Elastic Stack (aka ELK)].
```

Comment:
Replace this link: https://community.ibm.com/community/user/imwuc/viewdocument/on-demand-using-liberty-with-elast
with this link: https://developer.ibm.com/videos/send-open-liberty-logs-to-elastic-stack/